### PR TITLE
Add LL/CON/ADV/BV-06-C test case

### PR DESCRIPTION
**Pull request by Tronil**
Signed-off-by: Troels Nilsson <trnn@demant.com>

### CODE REVIEW

File: src/tests/ll_verification.py [hunk 1]
```diff
@@ -2401,6 +2401,206 @@ def wait_for_AUX_ADV_IND_end():
...
+    if not success:
+        return False
+
+    success = success and preamble_ext_advertise_enable(transport, upperTester, Advertise.ENABLE, [Handle], [0], [0], trace)
+    if not success:
+        return False
+
+    # Wait until the simulation time is just after an ADV_IND packet has transmitted (ie. within T_IFS so a response can be transmitted)
+    # Returns the ADV_IND packet or None if it fails to find one within an advertising interval
+    def wait_for_ADV_IND_end():
```
Comment (wopu-ot) [inline src/tests/ll_verification.py:2547]: Maybe this function could be moved to test_utils.py, next to `wait_for_AUX_ADV_IND_end`?

File: src/tests/ll_verification.py [hunk 2]
```diff
@@ -2401,6 +2401,206 @@ def wait_for_AUX_ADV_IND_end():
...
+        timeOut = math.ceil(advInterval*0.625+10)*1000
+        checkInterval = 100 # Note: Has to be less than T_IFS
+        advIndPacket = None
+        while timeOut > 0:
+            lastPacket = packets.findLast(packet_filter=('ADV_IND'))
+            if lastPacket:
+                # Check that simulation time is just after the ADV_IND has ended (so we can transmit a response)
+                simulation_time = transport.get_last_t()
+                airTime = (1 + 4 + 2 + len(lastPacket) + 3)*8
```
Comment (wopu-ot) [inline src/tests/ll_verification.py:2556]: I think `get_packet_air_time` could be used here

File: src/tests/ll_verification.py [hunk 3]
```diff
@@ -2401,6 +2401,206 @@ def wait_for_AUX_ADV_IND_end():
...
+        packetData = b''.join([packetData, 0x0018.to_bytes(2, 'little', signed=False)]) # LLData: Interval
+        packetData = b''.join([packetData, 0x0000.to_bytes(2, 'little', signed=False)]) # LLData: Latency
+        packetData = b''.join([packetData, 0x0019.to_bytes(2, 'little', signed=False)]) # LLData: Timeout
+        packetData = b''.join([packetData, 0x1FFFFFFFFF.to_bytes(5, 'little', signed=False)]) # LLData: ChM
+        packetData = b''.join([packetData, (0x07 + (0x05 << 5)).to_bytes(1, 'little', signed=False)]) # LLData: Hop and SCA (Hop: 7 and SCA: 5)
+        CRC = calcBLECRC(0x555555, packetData)
+        packetData = b''.join([packetData, CRC.to_bytes(3, 'little', signed=False)])
+
+        # Calculate transmit timestamp (T_IFS from end of ADV_IND)
+        airTime = (1 + 4 + 2 + len(advIndPacket) + 3)*8
```
Comment (wopu-ot) [inline src/tests/ll_verification.py:2589]: I think get_packet_air_time could be used here

**Review by wopu-ot — CHANGES_REQUESTED** [review]

File: src/tests/ll_verification.py (hunk 2 above)
Comment (Tronil) [reply src/tests/ll_verification.py:2556]: Yup - that function wasn't in yet when I made this. I'll rebase and use it instead

File: src/tests/ll_verification.py (hunk 1 above)
Comment (Tronil) [reply src/tests/ll_verification.py]: Done

**Pull request merged**